### PR TITLE
Lesson 9

### PR DESCRIPTION
**Pull request by alecxkelly**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,15 +3,39 @@
...
 </head>
 <body>
-    <h1>Alecxis</h1>
+<section id="About">  
+    <h1>Alecxiskelly.com</h1>
     <h2>About</h2>
-    <p>I am a creative Tech/Entrepreneur from Philadelhia, now I am worldwide.</p>
+    <p>I am a creative Tech/Entrepreneur from Philadelphia, now I am worldwide.</p>
+</section>
     <h2>Experience</h2>
```
Comment (a-zuniga) [inline index.html:19]: This should be wrapped in a ```<section></section>``` tag just how the about section is laid out!

File: index.html [hunk 2]
```diff
@@ -3,15 +3,39 @@
...
+            Portfolio Website
+        </li>
+        <li>
+            Kitchen Designer
+        </li> 
+        <li>
+            Product Designer
+        </li>
+   </ul> 
     <h2>Connect</h2>
```
Comment (a-zuniga) [inline index.html:31]: Should also be wrapped in a ```<section></section>``` tag

**Review by a-zuniga — CHANGES_REQUESTED** [review]
You're almost there! Just wrap the experience and connect elements with a sections tag and you'll meet all of the assignments requirements! Don't forget about the id as well, you did it correctly for the about section!

**Comment by TeresaSwarthout** [comment]
Lesson 9 looks good!  Great start on your site.

Few minor comments:
I am a creative Tech/Entrepreneur from Philadelphia, now I am worldwide.   ->  I'm not quite sure what this sentence means - you might expand on it a bit.
Your contact link doesn't work.
You have style comments here, which you will eventually want to put in your index.css
  body {background-color: powderblue;}
        h1 {color: rgb(5, 9, 223);}
        p {color: blue;}

**Pull request merged**